### DISCUSSION TITLE
Ignore character recognition

### DISCUSSION THREAD

**Discussion by mkoch66** [jomjol/AI-on-the-edge-device#3458] [Ideas] [answered]
Hello,

my gasometer values are from time to detected as N instead of the digit,  just like. 50N56.12N
Has this N a special a meaning? Or would it make sense to look on "previous Value" to guess the digit instead?

**Comment by SybexX** [comment, marked as answer]
"N" and "NaN" stands for "not correctly recognized" / "not a valid value"

https://jomjol.github.io/AI-on-the-edge-device-docs/FAQs/#how-accurate-are-the-detections
